tests/lib/hardware/hardware_base.py

```diff
@@ -195,6 +205,19 @@ def ansible_inventory_vars(self) -> Dict[str, Any]:
         }
         return vars
 
+    def get_node_by_role(self, role):
+        items = []
+        for node_name, node_obj in self.nodes.items():
+            if role in node_obj.tags:
+                items.append(node_obj)
+        return items
+
+    def get_masters(self):
+        self.masters = self.get_node_by_role('master')
```

Comment:
I already commented on that - can you please use the `NodeRole` enum value?

Reply:
Done